src/main/java/graphql/execution/ExecutionContext.java

```diff
@@ -33,15 +32,13 @@
     private final Object root;
     private final Object context;
     private final Instrumentation instrumentation;
-    //
-    // errors is kept in order via LinkedHashMap and thread safe via synchronised guards
-    private final Map<String, GraphQLError> errors = new LinkedHashMap<>();
+    private final List<GraphQLError> errors = new ArrayList<>();
```

Comment:
hi @bbakerman 
Why not to use [java.util.concurrent.CopyOnWriteArrayList](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CopyOnWriteArrayList.html) instead? 

No maintenance would be required to support/test thread synchronization.

Reply:
good catch